### PR TITLE
LPS-193167 Create a new blank workspace and update shell scripts

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [LPS-193167](https://issues.liferay.com/browse/LPS-193167).

### CODE REVIEW

**Comment by drewbrokke** [comment]
ci:test:sf

**Comment by drewbrokke** [comment]
ci:test:relevant

**Comment by drewbrokke** [comment]
ci:forward

File: workspaces/liferay-blank-workspace/configs/local/portal-ext.properties [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#
+# MySQL
+#
+#jdbc.default.driverClassName=com.mysql.cj.jdbc.Driver
+#jdbc.default.url=jdbc:mysql://localhost/lportal?useUnicode=true&characterEncoding=UTF-8&useFastDateParsing=false
+#jdbc.default.username=root
+#jdbc.default.password=
+
+feature.flag.LPS-164563=true
```
Comment (gamerson) [inline workspaces/liferay-blank-workspace/configs/local/portal-ext.properties:9]: Do we want this for all new workspaces done by blade init?  To enable all  of these CX related feature flags?

File: workspaces/liferay-blank-workspace/gradle.properties [hunk 2]
```diff
@@ -0,0 +1,12 @@
+##
+## Check GETTING_STARTED.markdown for additional properties and their default
+## values.
+##
+
+liferay.workspace.bundle.dist.include.metadata=true
+liferay.workspace.modules.dir=modules
+liferay.workspace.node.package.manager=yarn
```
Comment (gamerson) [inline workspaces/liferay-blank-workspace/gradle.properties:8]: Are we wanting to make yarn the new default for new `blade init` workspaces?

**Comment by gamerson** [comment]
In general the rest of this looks good.

**Comment by drewbrokke** [comment]
ci:forward

File: workspaces/liferay-blank-workspace/gradle.properties (hunk 2 above)
Comment (drewbrokke) [reply workspaces/liferay-blank-workspace/gradle.properties:8]: I think we can parameterize this one, or strip it out by default.

File: workspaces/liferay-blank-workspace/configs/local/portal-ext.properties (hunk 1 above)
Comment (drewbrokke) [reply workspaces/liferay-blank-workspace/configs/local/portal-ext.properties:9]: We'll probably strip out the flags during blade init.

**Pull request closed without merging**